### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/security-and-auditing-full-course-s23#119] [General]
hi guys, as long as there are tools like aderyn and slither, it's easily identify all kinds of bugs, what is the need for new auditor?

**Comment by alymurtazamemon** [comment]
@itsmohammadh These tools can only catch low-hanging common bugs, but in real projects, you will see bugs in the business logic which these tools cannot catch and therefore security researchers are required to fulfil the need.
